spark/src/test/java/com/matthewrathbone/sparktest/SparkJoinsTest.java

```diff
@@ -0,0 +1,74 @@
+package test.java.com.matthewrathbone.sparktest;
```

Comment:
Is there a reason you decided to use the Java API instead of the native scala api? I'm not sure how many folks actually use the java version.